src/cursor/cur_file.c

```diff
@@ -640,8 +640,10 @@ __curfile_close(WT_CURSOR *cursor)
     dead = F_ISSET(cursor, WT_CURSTD_DEAD);
 
     /* Free the bulk-specific resources. */
-    if (F_ISSET(cursor, WT_CURSTD_BULK))
+    if (F_ISSET(cursor, WT_CURSTD_BULK)) {
         WT_TRET(__wt_curbulk_close(session, (WT_CURSOR_BULK *)cursor));
```

Comment:
Let me know if you think I should hide the stat update in `__wt_curbulk_close` instead. It might be better if it's used somewhere else in the code at some point?